File: src/test/java/cernx/colt/matrix/tfloat/impl/SparseCCFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class SparseCCFloatMatrix2DViewTest extends SparseCCFloatMatrix2DTest {

    public SparseCCFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}

File: src/test/scala/cern/colt/matrix/tfloat/impl/SparseCCFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.SparseCCFloatMatrix2D

class SparseCCFloatMatrix2DViewTest(arg0: String) extends SparseCCFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseCCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseCCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseCCFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
